include/picongpu/simulation_types.hpp

```diff
@@ -34,16 +34,14 @@
 
 namespace picongpu
 {
-    //! define all elements which can send and resive
-
+    /** Legacy communication tags
+     *
+     * Now only used for some field traits, to be removed soon
+     */
     enum CommunicationTag
```

Comment:
Can we rename this type to something else, it is now only used to get the margins. 
As your new doxygen says it is no MPI communication tag anymore: https://github.com/ComputationalRadiationPhysics/picongpu/pull/3558/files#diff-ef03a551bc4457d540a9f7b8ee58246c94e3014958928ac33da6fd353a3c0d1dR82

It is something like an alias for the buffer/field. Maybe we should name it `fieldAlias`?

Reply:
I will remove it in the next PR after this is merged

Reply:
I am fine to move it into a follow up PR